CONTRIBUTING.md

```diff
@@ -53,7 +53,22 @@ about Git.
   - Create pull request
 
 The title of the pull request should be prefixed by the component or area that
-the pull request affects. Examples:
+the pull request affects. Valid areas as:
+
+  - *Consensus* for changes to consensus critical code
```

Comment:
That's not correct, where they are prefixed, it is usually with brackets - but most PRs don't use prefixes which is why you dont see them mostly.